pkg/logging/logging.go

```diff
@@ -0,0 +1,50 @@
+package logging
+
+import (
+	"io"
+	"os"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+const EnvLevel = "LOG_LEVEL"
+
+type Config struct {
+	Structured     bool
+	Level          Level
+	LevelFieldName string
+}
+
+type Level struct {
+	zerolog.Level
+}
+
+// Set the level using level or LOG_LEVEL environment variable
+// return an error if the level cannot be parsed.
+func (l *Level) Set(level string) error {
+	if level == "" {
```

Comment:
No, this should set the `LOG_LEVEL` to `NoLevel` https://godoc.org/github.com/rs/zerolog#Level

Reply:
Done, debug by default
Removed this function since flag.parse only calls it if a level is set using the more usual `StringVar` with `BuildDefaultConfigItem`